scripts/variantstore/scripts/hail_create_vat_inputs.py

```diff
@@ -3,7 +3,7 @@
 from typing import Union
 from datetime import datetime
 
-import create_vat_inputs
+from scripts.variantstore.scripts import create_vat_inputs
```

Comment:
is this correct? I wouldn't have expected that `scripts.variantstore` bit

Reply:
Hmm. Good find. I haven't tested creating the vat - guessing that would fail?

Reply:
Okay, that *was* incorrect. Working now.